src/Illuminate/Database/Grammar.php

```diff
@@ -63,6 +63,14 @@ public function wrap($value, $prefixAlias = false)
             }
 
             return $this->wrap($segments[0]).' as '.$this->wrapValue($segments[2]);
+        } elseif (strpos(strtolower($value), ' ') !== false) {
```

Comment:
@taylorotwell we first check if the table (from) value has `as` alias on first if condition. 
```sql
select u.* from users as u;
```

The second condition catches table alias declaration without `as` like:

```sql
select u.* from users u;
```
This catches the changes made on BelongsTo class, etc and fixes the unit tests.

Thanks!